src/parser/class.js

```diff
@@ -226,13 +226,17 @@ module.exports = {
   /*
    * Reads constant list
    * ```ebnf
-   *  constant_list ::= T_CONST (constant_declaration ',')* constant_declaration
+   *  constant_list ::= T_CONST [type] (constant_declaration ',')* constant_declaration
    * ```
    */
   read_constant_list: function (flags, attrs) {
     if (this.expect(this.tok.T_CONST)) {
       this.next();
     }
+
+    const [nullable, type] =
+      this.version >= 830 ? this.read_optional_type() : [false, null];
```

Comment:
I believe PHP 8.3 translate to version "803". See [this comment ](https://github.com/glayzzle/php-parser/blob/main/src/index.js#L49) and the [implementation of version parsing](https://github.com/glayzzle/php-parser/blob/main/src/index.js#L87).

Reply:
Indeed, thanks for pointing that out! Fix: https://github.com/glayzzle/php-parser/pull/1149